chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/validation/DeveloperActiveValidation.java

```diff
@@ -2,21 +2,37 @@
 
 import org.springframework.stereotype.Component;
 
+import gov.healthit.chpl.changerequest.domain.ChangeRequest;
 import gov.healthit.chpl.dto.DeveloperDTO;
 import gov.healthit.chpl.entity.developer.DeveloperStatusType;
+import gov.healthit.chpl.exception.EntityRetrievalException;
 import gov.healthit.chpl.manager.rules.ValidationRule;
 
 @Component
 public class DeveloperActiveValidation extends ValidationRule<ChangeRequestValidationContext> {
 
     @Override
     public boolean isValid(ChangeRequestValidationContext context) {
-        // Does it exist in the DB?
+
+        // Is this a new or existing CR?
+        ChangeRequest crToTest = null;
+        if (context.getChangeRequest() != null && context.getChangeRequest().getId() != null) {
+            try {
+                crToTest = context.getChangeRequestDAO().get(context.getChangeRequest().getId());
+            } catch (EntityRetrievalException e) {
+                // This should be caught be ChangeRequestExistenceValidation
+                return true;
+            }
+        } else {
+            getMessages().add(getErrorMessage("changeRequest.developer.notActive"));
```

Comment:
Is this the right error message for when context.getChangeRequest is null? The "if" part doesn't seem to have to do with the status of the developer, but I could be missing something!